src/nxdoc/web-ui/web-ui-overview.md

```diff
@@ -106,41 +105,38 @@ version_override:
 
 ![]({{file name='web_ui_with_side_panel.png'}} ?w=650,border=true)
 
-It leverages the web as a platform and allows enhanced productivity with direct access to last visited documents, collections, easy browsing between previous/next document, full-screen suggester and several other features built to provide a rich user experience. Here are the most important topics on getting started with Web UI, customizing it and building new web apps using Nuxeo Elements:
+It leverages the web as a platform and allows enhanced productivity with direct access to last visited documents, collections, easy browsing between previous/next document, full screen suggester and several other features built to provide a rich user experience. Here are the most important topics on getting started with Web UI, customizing it and building new web apps using Nuxeo Elements:
+
+<div class="row"><div class="column medium-4">{{#> panel type='primary'}}
 
-<div class="row">
-<div class="column medium-4" align="center">
-{{#> panel type='primary'}}
 [Polymer Guide]({{page page='polymer-guide'}})
-{{/panel}}
-</div>
-<div class="column medium-4" align="center">
-{{#> panel type='primary'}}
+
+{{/panel}}</div><div class="column medium-4">{{#> panel type='primary'}}
+
 [Nuxeo Elements]({{page page='nuxeo-elements'}})
-{{/panel}}
-</div>
-<div class="column medium-4" align="center">
-{{#> panel type='primary'}}
+
+{{/panel}}</div><div class="column medium-4">{{#> panel type='primary'}}
+
 [Customization]({{page page='web-ui-customization'}})
-{{/panel}}
-</div>
-</div>
+
+{{/panel}}</div></div>
 
 {{#> callout type='info' heading='Nuxeo University'}}
-Watch the related courses on Nuxeo University:</br>
-[Course on the Web UI Stack](https://university.nuxeo.com/learn/public/course/view/elearning/76/nuxeo-web-ui-stack-introduction).</br>
-[Course to Start with Web UI](https://university.nuxeo.com/learn/public/course/view/elearning/92/getting-started-with-nuxeo-web-ui).
+Watch the related courses on Nuxeo University
+- [Course on the Web UI Stack](https://university.nuxeo.com/learn/public/course/view/elearning/76/nuxeo-web-ui-stack-introduction).
```

Comment:
These links need to be updated because they aren't valid anymore, they redirect the user to the University homepage.